src/main/core/Resources/config/services/event.yml

```diff
@@ -0,0 +1,21 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Claroline\CoreBundle\Event\Log\AddRoleEvent: ~
+
+    Claroline\CoreBundle\Event\Log\RemoveRoleEvent: ~
+
+    Claroline\CoreBundle\Event\Log\UserLogoutEvent: ~
+
+    Claroline\CoreBundle\Event\Log\UserLogineEvent: ~
+
+    Claroline\CoreBundle\Event\Log\UserEnableEvent: ~
+
+    Claroline\CoreBundle\Event\Log\UserDisableEvent: ~
+
+    Claroline\CoreBundle\Event\Log\NewPasswordEvent: ~
+
+    Claroline\CoreBundle\Event\Log\ForgotPasswordEvent: ~
```

Comment:
events are not services :) this file can be removed

Reply:
Yeah sorry ^^"